### ISSUE TITLE
Update to Cars REPO

### ISSUE THREAD

**Issue by TomQD-94** [epics-modules/xspress3#44]
@newville When the master branch gets updated the method to install is not updated in relation to it. i.e the xspress3-2-x folder that is downloaded by the build script does not have updated files. Is there a way to automate this so that any master branch update gets put put into the repo and the file pulled by the build script is incremented?

**Comment by newville** [comment]
@TomQD-94 right, at the moment those come from releases (and me copying tarballs from github, so "explicit publishing"), not the current master.    I think the main thing to do is make a new release.   Do we think we are ready for that?  I would say yes.

We could update `build_xspress3.py` to fetch the tarball from github...

**Comment by TomQD-94** [comment]
@newville I would say this is good for a new release. Perhaps there should be an option to pull it from github otherwise a simple bugfix would cause you to set up a new release, new tarball, and then update the script. Although I don't know how often new releases will happen

**Comment by newville** [comment]
@TomQD-94 OK, tagged and released 3.2.7.  The latest `build_xspress3.py` should work, but I have not tested that yet ;).
